docs/users.rst

```diff
@@ -1202,6 +1202,8 @@ tagging the resulting image in the registry:
   over time. In other words, every time you build a new container image,
   OSBS updates these floating tags. Examples: ``latest``, or ``{version}``
 
+  The floating tags must match the pattern `^[\w.]{0,127}$`. 
+  
```

Comment:
A concrete example is always better than giving such a regular expression in documentation. We can't assume a user is familiar or even master the regular expression, and this regex is not much straightforward for a user to have a quick understand what a floating tag could look like. The unique tag example mentioned above is a good example to follow I think.

Reply:
While I don't think this should block merging this PR (as it already improves the situation - where the docs provide neither a concrete example, nor the validation rule), I'll go ahead and add one. ;)

Reply:
@tkdchen resolving this after adding an example, please reopen if you need something further.